src/NzbDrone.Core/IndexerSearch/NzbSearchService.cs

```diff
@@ -158,6 +166,18 @@ public List<DownloadDecision> SeasonSearch(int seriesId, int seasonNumber, bool
             return downloadDecisions;
         }
 
+        public List<DownloadDecision> AlbumSearch(int albumId, bool missingOnly, bool userInvokedSearch)
+        {
+            var album = _albumService.GetAlbum(1);
```

Comment:
At the moment `albumId` is always being passed as 0. Once I get the release processing handled, I'll return to this to try to dig up why it's always 0.